modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLCopyEntryDisplayContext.java

```diff
@@ -225,6 +221,7 @@ private long _getSourceRepositoryId() throws PortalException {
 		return repository.getRepositoryId();
 	}
 
+	private String _copyEntryURL;
```

Comment:
This should be `_copyDLObjectURL` for consistency with the parameter name. The method should probably be named like that as well.